### PR TITLE
fix a typo in configuration key constant

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

**Comment by Ladicek** [comment]
I found this typo by accident, and didn't investigate much whether this change is safe as it is, or whether it requires further changes. I did grep the entire codebase for `clinet` and this seems to be the only occurence, so it's possible this change is sufficient (hence this PR) -- but I'll defer to your judgement, as I'm not really familiar with the code.

**Comment by Ladicek** [comment]
Ah one more thing -- such typo is not a big deal if you do configuration programmatically, but once you have to use the strings (e.g. with Quarkus and SmallRye Reactive Messaging), it starts to matter quite a bit. I know you plan to do a 2.0.0 release soon(-ish), so if this could be merged before, that would be great :-)

**Comment by carlesarnal** [comment]
Good catch @Ladicek, yes, it's a typo. I will wait for CI but looks good.

**Review by carlesarnal — APPROVED** [review]

**Pull request merged**
